src/modules/m_joinflood.cpp

```diff
@@ -199,7 +203,7 @@ class ModuleJoinFlood
 			{
 				f->clear();
 				f->lock();
-				memb->chan->WriteNotice(InspIRCd::Format("This channel has been closed to new users for %u seconds because there have been more than %d joins in %d seconds.", duration, f->joins, f->secs));
+				memb->chan->WriteNotice(InspIRCd::Format(notifymessage.c_str(), duration, f->joins, f->secs));
```

Comment:
This could break pretty easily by changing or removing the format specifiers in the configuration, that's not good. For now, I would recommend removing the changes regarding this message and just keep the changes for the user who tried to join.

Reply:
(This is why I haven't merged it yet fwiw.)

Reply:
Removed notifymessage in code and docs.